NSS-Day-Cohort-49/giffygram-double-rainbows
# Delete like

#### Changes Made

1. Modified file `PostFeed.js` to include send the likeId instead of the postId
​
#### Steps to Review
1. Checkout this branch locally.
    ```
    git fetch --all
    git checkout delete-likes
    ```
2. Open a new Terminal tab (⌘T) and navigate to the server directory.
3. Test app functionality.
    > Instructions for how reviewer can test functionality, and detailed description of what the expected outcome is.
    > Example: When user does hits the like button it added or deletes the like
4. View code file.
    > Confirm file modifications are present as indicated above.
    > Confirm no unused code or extraneous comments exist.
